sphinx_gallery/tests/test_gen_gallery.py

```diff
@@ -211,6 +211,16 @@ def test_duplicate_files_warn(sphinx_app_wrapper):
     assert msg.format(m) in build_warn
 
 
+@pytest.mark.conf_file(content="""
+sphinx_gallery_conf = {
+    'rebuild': 'wrong_value',
+}""")
+def test_config_rebuild(sphinx_app_wrapper):
+    """Test ValueError is raised for invalid rebuild values"""
+    with pytest.raises(ValueError, match="The 'rebuild' parameter must be"):
+        sphinx_app_wrapper.create_sphinx_app()
```

Comment:
You can use the `tinybuild` infrastructure in `test_full.py` to do two builds in a row, ensuring that no examples are run the second time around.